crates/trie/trie/src/hashed_cursor/post_state.rs

```diff
@@ -179,34 +179,33 @@ where
 /// The cursor to iterate over post state hashed storages and corresponding database entries.
 /// It will always give precedence to the data from the post state.
 #[derive(Debug, Clone)]
-pub struct HashedPostStateStorageCursor<'b, C> {
+pub struct HashedPostStateStorageCursor<'a, C> {
     /// The database cursor.
     cursor: C,
     /// The reference to the post state.
-    post_state: &'b HashedPostStateSorted,
-    /// The current hashed account key.
-    hashed_address: B256,
+    post_state_storage: Option<&'a HashedStorageSorted>,
```

Comment:
do we need to update naming here? because this is now `HashedStorage` not `HashedPostState`

Reply:
the name was updated to `post_state_storage`

Reply:
i'll update the doc